### DISCUSSION TITLE
I'm unable to verify your academic affiliation.

### DISCUSSION THREAD

**Discussion by sbendarsky** [community/community#56774] [GitHub Education]
### Hi everyone,

I've been trying to verify my academic affiliation in the past few days, and I've been attempting different methods. I obtained a student ID, but it wasn't accepted because it wasn't in English. I then used Google Translate to translate my name and added both versions to the picture, but it was flagged as "Manipulated images may not be used." I also got an official transcript from my college stating that I am currently studying, but it wasn't accepted either. I'm in need of some help to figure out what to do next.

**Comment by Raja-Debnath** [comment]
Same facing some issues here , noticed back camera not working 😕, so I want about to verify, as when ever I uploaded it said to use camera for verification

**Reply by Raja-Debnath** [reply]
Done , i used a droidcam app to make my mobile as a camera and did the verification

**Comment by Sandrakay49** [comment]
Name state city CPS workers ASAP

**Reply by Sandrakay49** [reply]
WV CPS workers I need that because I know they was the one and I need their name and everything and I needed it ASAP

**Comment by Sandrakay49** [comment]
Check and see if any CPS workers out of Cabll county or Charleston West Virginia

**Comment by Raja-Debnath** [comment]


**Reply by Sandrakay49** [reply]
So you can't find anything
